src/v2/Apps/Settings/Routes/EditProfile/UserInformation.tsx

```diff
@@ -74,11 +73,18 @@ export const UserInformation: React.FC<UserInformationProps> = ({
 
   return (
     <Box>
-      <Serif size="6" mb={space(2)}>
+      <Text size="6" mb={2}>
         Information
-      </Serif>
+      </Text>
       <Formik
-        initialValues={me}
+        initialValues={
+          {
+            name: me.name,
+            email: me.email,
+            phone: me.phone,
+            paddleNumber: me.paddleNumber
+          }
+        }
         validationSchema={ChangeUserInformationValidator}
         onSubmit={onSubmit}
```

Comment:
We'll want to use the `useToasts` hook to send update/error statuses on success/failure

Reply:
Updated and added in the `useToasts` hooks